src/main/java/nl/tudelft/semgroup4/GameEndedState.java

```diff
@@ -22,208 +20,406 @@
-                typeFontPlayer.getHeight());
-        
+                Integer.toString(players[0].getScore()), Color.white);
+        typeFontPlayer.drawString(container.getWidth() / 5 * 3,
+                container.getHeight() / 3,
+                Integer.toString(players[0].getMoney()), Color.white);
+
     }
-    
+
     /**
-     * Setup for initializing extra data after the state has been made at the beginning
-     * of the app.
-     * @param players Players currently in the game
-     * @param won True if the game has been won
+     * draws the seconds player information onto the screen.
+     * 
+     * @param container
+     *            container in which to draw the information
+     * @param graphics
+     *            graphics to use to draw the elements
      */
-    public void setup(Player[] players, boolean won) {
-        this.won = won;
-        this.players = players;
+    public void drawSecondPlayerInfo(GameContainer container, Graphics graphics) {
+        typeFontPlayer.drawString(container.getWidth() / 8,
+                container.getHeight() / 2, "PLAYER 2", Color.white);
+        typeFontPlayer.drawString(container.getWidth() / 5 * 2,
+                container.getHeight() / 2,
+                Integer.toString(players[1].getScore()), Color.white);
+        typeFontPlayer.drawString(container.getWidth() / 5 * 3,
+                container.getHeight() / 2,
+                Integer.toString(players[1].getMoney()), Color.white);
+
+        // Draws the second players textfield box.
+        textFieldPlayer2.render(container, graphics);
+
+        // Draws a white rectangle around the textfield because
+        // slick can't handle it on it's own.
+        graphics.drawRect(container.getWidth() / 5 * 4,
+                container.getHeight() / 2, textFieldPlayer1.getWidth() - 1,
+                textFieldPlayer1.getHeight() - 1);
     }
 
-    @Override
-    public void render(GameContainer container, StateBasedGame game, Graphics graphics)
-            throws SlickException {
-        //draws a different title depending on if the game has been won or lost.
+    /**
+     * Draws the title according to of the game has been won or lost.
+     * 
+     * @param container
+     *            container in which to draw the title
+     */
+    public void drawTitle(GameContainer container) {
         if (won) {
             typeFontTitle.drawString(
-                    container.getWidth() / 2 - typeFontTitle.getWidth(gameOverText) / 2,
+                    container.getWidth() / 2
+                            - typeFontTitle.getWidth(gameOverText) / 2,
                     container.getHeight() / 8, gameWonText, Color.yellow);
         } else {
             typeFontTitle.drawString(
-                    container.getWidth() / 2 - typeFontTitle.getWidth(gameOverText) / 2,
+                    container.getWidth() / 2
+                            - typeFontTitle.getWidth(gameOverText) / 2,
```

Comment:
You use the gameOverText in both branches of your if-statement. 
Also, I think it would be better to just use one variable for this, which you give one of the two possible texts. 
And to get this text, you should use a controller instead of this class, to keep as little logic as possible in this class.